src/main/scala/mix/kafka/monitor/MetricsCacheProvider.scala

```diff
@@ -0,0 +1,57 @@
+package mix.kafka.monitor
+
+import java.time.Duration
+import java.util.concurrent.TimeUnit
+
+import com.codahale.metrics.MetricRegistry
+import kafka.utils.Logging
+import mix.kafka.monitor.LibratoOffsetReporter.getMetricName
+
+case class GroupTopic(group: String, topic: String)
```

Comment:
Since `GroupTopic` is used across classes, either move it to a separate file or add it to a package object.